scripts/new-version-pr-generator.js

```diff
@@ -0,0 +1,154 @@
+/* eslint-disable no-console */
+const {execSync} = require('child_process');
+const {readFileSync} = require('fs');
+const pathResolve = require('path').resolve;
+const {mkdir} = require('shelljs');
+const yaml = require('js-yaml');
+const semver = require('semver');
+
+const {version: PACKAGE_VERSION} = require('../package.json');
+const secrets = require('../secrets.json');
+const Retry = require('./utilities/retry');
+
+const repositories = ['polaris-styleguide', 'web'];
+
+const YARN_VERSION = yaml
+  .safeLoad(readFileSync(pathResolve(__dirname, '..', 'dev.yml'), 'utf8'))
+  .up.find((obj) => Object.keys(obj).includes('node')).node.yarn;
+
+const root = pathResolve(__dirname, '../');
+const sandbox = pathResolve(root, 'sandbox');
+
+const releaseVersion = `v${PACKAGE_VERSION}`;
+
+function isMajorPrerelease(version) {
+  return (
+    semver.prerelease(version) &&
+    semver.minor(version) === 0 &&
+    semver.patch(version) === 0
+  );
+}
+
+const baseBranch = isMajorPrerelease(PACKAGE_VERSION)
+  ? `v${semver.major(PACKAGE_VERSION)}`
+  : 'master';
+
+const polarisBotName = 'Shopify Polaris Bot';
+const polarisBotEmail = 'shopify-polaris-bot@users.noreply.github.com';
+const polarisBotToken = secrets.github['shopify-polaris'];
+const shopifyPolarisBotGitOverride = `GIT_COMMITTER_NAME='${polarisBotName}' GIT_COMMITTER_EMAIL='${polarisBotEmail}'`;
+
+function pullRequestTemplate(repository, version) {
+  return `
+## Version ${version} of @shopify/polaris just got published!
+
+See what’s new: https://github.com/Shopify/polaris-react/releases/tag/${version}
+
+cc @kaelig @dfmcphee @amrocha @BPScott
+
+---
+
+<details>
+<summary>🚨 What to do if you see “Your tests failed on CircleCI”?</summary>
+
+If tests fail, you may have to troubleshoot the problem locally.
+
+1. Checkout the \`update-polaris-${version}\` branch:
+    \`\`\`bash
+    dev cd ${repository}
+    git fetch
+    git checkout update-polaris-${version}
+    dev up
+    \`\`\`
+1. Apply changes/fixes
+1. Commit
+1. Push your work to the \`update-polaris-${version}\` branch:
+    \`\`\`bash
+    git push origin update-polaris-${version}
+    \`\`\`
+1. Repeat until the build & tests go ✅
+1. 🎩
+1. :shipit:
+</details>
+`.trim();
+}
+
+function failedUpdateMessage(repository, version) {
+  return `It seems your automatic branch creation for ${repository} failed. This can be due to many reasons. To resolve this follow these steps:
```

Comment:
I'd replace "your" with something less personal, as the person reading the logs may not have triggered the build themselves.